frontends/web/src/components/rates/rates.module.css

```diff
@@ -59,7 +59,7 @@
 }
 
 @media (max-width: 640px) {
-    .availableFiatUnit::after {
+    .rotatable::after {
         border-bottom: none;
     }
```

Comment:
It seems it was added in this old PR

https://github.com/digitalbitbox/bitbox-wallet-app/pull/452/files#diff-e1e18eb9c98cc4f9b2d8c9920ef51acdc9a87c3ff21c67bf370394c2f5797934R49-R53

I can't think of a good reason other than for visual reasons... But when I remove the border-bottom none it is hard to see, maybe the reason back then was that since it almost not visible we might as well remove it completely?

Reply:
at least in dark mode it is very hard to see. this could be definetly improved, but maybe out of scope for this PR?

Reply:
please remove this rule so it shows also on mobile. even if it is hard to see. there is no good reason to hide it.